commonjs/view-ajax/view.js

```diff
@@ -246,6 +246,7 @@ ViewAjax.prototype = {
         this._getState().searchFormValues = this.searchForm.getValues();
         var url = location.protocol+'//'+location.host+this.viewUrl+'?'+$.param(this.searchForm.getValuesIncludingPost());
         historyState.pushState(document.title, url);
+        this.$el.trigger('search', this._getState().searchFormValues);
```

Comment:
- wär das nicht weiter oben wo this.load() aufgerufen wird richtiger (da ist auch ein 500ms buffer)
- ich würde dieses event kokreter benennen

Reply:
Ist verschoben und umbenannt